### DISCUSSION TITLE
Exporting documents fails with Status 404, error "not_found"

### DISCUSSION THREAD

**Discussion by SuperSonnix71** [outline/outline#6840] [Self Hosting]
Hi everyone.
I'm trying to export documents from our Wiki  using "api/documents.export" and i keep getting "not found" no matter how i try. I would appraciate all the help. 

for example:
I can retrieve a specific document information using "documents.info"
{
    "data": {
        "id": "fd432eca-6c80-4206-8d7e-ea3391c4a95c",  <------------------- Using this as document ID for export
        "url": "/doc/XXX-documentation-5UgIUXXs1C",
        "urlId": "5UgIUXXs1C",
        "title": "XXX Documentation",
        "emoji": null,
        "text": " ![](https://wiki.XXX.com/outline-bucket/uploads/505469d2-d49a-4e39-81a3-991ac9c49dc3/72f8f2ac-e3d9-4be4-aba2-d8850a1b3a5a/image.png?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=dd96e906204c9e7f%2F20240424%2Fxx-xxxx-x%2Fs3%2Faws4_request&X-Amz-Date=20240424T045951Z&X-Amz-Expires=3000&X-Amz-Signature=be0c2bdb60108864efc66c085a2449bcd1e1c99ec3a720874669f953d4d63994&X-Amz-SignedHeaders=host&response-content-disposition=attachment)",
        "tasks": {
            "completed": 0,
            "total": 0
        },
        "createdAt": "2021-06-15T09:50:54.318Z",
        "updatedAt": "2024-03-21T11:09:24.667Z",
        "publishedAt": "2021-06-15T09:51:04.755Z",
        "archivedAt": null,
        "deletedAt": null,
        "teamId": "6b4126ab-0cf7-4d2b-9cab-7944bfaddaf4",
        "collaboratorIds": [],
        "revision": 5,
        "fullWidth": false,
        "isCollectionDeleted": false
    },
    "status": 200,
    "ok": true
}
~$ curl --request POST \
    --url https://XXX.XXX.com/api/documents.export \
    --header 'Authorization: Basic XXXXXXXXXXXXXXXXXXXXXXXXXX==' \
    --header 'content-type: application/json' \
    --data '{
  "id": "fd432eca-6c80-4206-8d7e-ea3391c4a95c"
  
}'
{"ok":false,"error":"not_found","status":404,"message":"Resource not found"}

**Comment by tommoor** [comment]
It needs to be a POST request
